moveit_experimental/jog_arm/src/jog_arm/teleop_examples/spacenav_to_twist.cpp

```diff
@@ -0,0 +1,66 @@
+#include "geometry_msgs/TwistStamped.h"
+#include "moveit_msgs/JogJoint.h"
+#include "ros/ros.h"
+#include "sensor_msgs/Joy.h"
+
+namespace jog_arm
+{
+class SpaceNavToTwist
+{
+public:
+  SpaceNavToTwist() : spinner_(1)
+  {
+    joy_sub_ = n_.subscribe("spacenav/joy", 1, &SpaceNavToTwist::joyCallback, this);
```

Comment:
make the constant ``1`` a constant variable with useful variable name so we know what it represents (buffer size?)

Reply:
same on next two lines